lte/gateway/c/session_manager/sessiond_main.cpp

```diff
@@ -358,7 +364,7 @@ int main(int argc, char* argv[]) {
       if (config["poll_stats_interval"].IsDefined()) {
         interval = config["poll_stats_interval"].as<uint32_t>();
       }
-      periodic_stats_requester->start_loop(local_enforcer, interval);
+      periodic_stats_requester->start_loop(local_enforcer, interval, shard_ids);
```

Comment:
if you do it this way then shard_ids will always be empty. You have to iterate over the shard tracker class you have